bin/.gitignore

```
@@ -1 +1,6 @@
+<<<<<<< Updated upstream
 .classpath
+=======
+/events/
+/states/
+>>>>>>> Stashed changes
```

I'm not sure what this stuff is denoting? If it's from a merge should it be deleted?

---

I hope we won't mess anything up by deleting this. (Stu**d gitignore...) I can delete it though.

---

Def don't delete the whole file. Only the merge stuff like:
```suggestion
.classpath

```